### PR TITLE
Updated copy

### PR DESCRIPTION
**Pull request by bpdesigns**
Copy ready for review. Added some notes and links about 508 and accessibility. I've reached out to folks over slack to see if we've used the IAA with an agency yet. The money is set aside but no research has been conducted and paid for through it. We should come back to the compensation section after we have more examples of how people are paying for recruitment and if we get new guidance from OGC.

### CODE REVIEW

**Comment by jstrothman** [comment]
@bpdesigns Could you take a look at my edits? There's enough that I don't want to approve my own commit.
I moved the compensation to the Plan page, and made some content edits -- some plain language and some to emphasize participant control over stopping sessions.

File: _pages/research/do.md [hunk 1]
```diff
@@ -120,13 +120,14 @@ An interview guide keeps sessions consistent, allows multiple people to moderate
...
-    - Remind participants that you are not testing them, and that there are no wrong answers
-    - Remind participants that their participation does not entitle them to compensation (unless compensation was offered during recruitment), and they can decide at any point that they no longer want to participate
-- Consent for note-taking or recording
-    - Tell participants how you’ll use the notes
-    - Tell participants whether you’ll attribute any quotes, and if so, how (by name or by role, for example)
-    - Confirm that participants have signed a consent form; get verbal consent to proceed
+- Introductory preamble covers:
+    - You are not testing them, and that there are no wrong answers
+    - Explain whether their participation qualifies them for [compensation]({{site.baseurl}}/research/plan/#compensating-research-participants) (and if so discuss how/when they will receive it)
+    - They are in control of their participation: they can tell you if they'd prefer not to answer a question, they can take a break any time, and they can decide at any point that they no longer want to participate
```
Comment (bpdesigns) [inline _pages/research/do.md:126]: I would also say something like... "if the participant qualifies for compensation, make it clear that if they decide they do not want to answer a question or end their participation early, they will still receive the agreed upon compensation."

**Review by bpdesigns — COMMENTED** [review]
@jstrothman Looks good overall! I added one suggestion.

**Review by jstrothman — COMMENTED** [review]
looking good!

**Pull request merged**